src/TaskList.js

```diff
@@ -1,14 +1,39 @@
 class TaskList {
-  static #taskList = [
-    { description: 'aaa', completed: true, index: 1 },
-    { description: 'bbb', completed: true, index: 2 },
-    { description: 'ccc', completed: false, index: 3 },
-    { description: 'ddd', completed: false, index: 4 },
-  ];
-
-  static getTaskList() {
-    return this.#taskList;
+  static #taskList = [];
+
+  static #lsTaskList = 'taskList';
+
+  static getTaskList = () => TaskList.#taskList;
+
+  static setStore() {
+    localStorage.setItem(TaskList.#lsTaskList, JSON.stringify(TaskList.#taskList));
+  }
+
+  static getStore() {
+    TaskList.#taskList = JSON.parse(localStorage.getItem(TaskList.#lsTaskList));
   }
+
+  static add = (task) => {
+    const index = TaskList.#taskList.length + 1;
+    const newTask = { ...task, completed: false, index };
+    TaskList.#taskList.push(newTask);
+    TaskList.setStore();
+    return newTask;
+  };
+
+  static get = (index) => TaskList.#taskList[index - 1];
```

Comment:
@DeliceLydia 
Hello, I do not understand, what you mean by not being able to do anything... I have no idea on how you are testing.
I believe the problem has to do with the deploy, and not with the code, because when I do
- npm install
- npm run start

It opens a tab in the web browser and everything runs as expected.

Would you be kind to explain what you are doing to test, so that I can understand what is the problem?
Is it possible to have a zoom call?

Thank you.

Reply:
@rbanos-mv Thanks for reaching out actually Server is running smoothly but I am not able to do anything

Reply:
@DeliceLydia
Your explanation is so short... I cannot guess what you mean by that... would you please explain what are you doing to test the project?